dev-server/static/scripts/util.js

```diff
@@ -19,7 +19,9 @@ export function loadClient(clientUrl) {
  * This uses the same method as the browser extension does to deactivate the client.
  */
 export function unloadClient() {
-  const annotatorLink = document.querySelector('link[type="application/annotator+html"]');
-  annotatorLink?.dispatchEvent(new Event('destroy'));
+  const annotatorLink = document.querySelector('link[type="application/annotatror+html"]');
+  // Do not use optional chaining (?.) below. Because this code is not bundled by babel
+  // we need to make compatible with older browsers 
+  annotatorLink && annotatorLink.dispatchEvent(new Event('destroy'));
```

Comment:
```suggestion
if (annotatorLink) {
  annotatorLink.dispatchEvent(new Event('destroy'));
}
```

What you have written works and probably what something like Terser would generate, but I think using a conditional is more idiomatic if we're restricted to not using optional chaining.